File: AlgorithmsToolbox/Week1/APlusB.cpp
#include <iostream>

int main(){
  int a = 0;
  int b = 0;
  int sum = 0;
//  std::cin >> a;
//  std::cin >> b;
	std::cin >> a >> b; 

	if(a >=0 && b<=9)
	{
  	sum = a + b;
  	std::cout << sum << std::endl;
	}

	else
		std::cout << "error in inputs" << std::endl;

  return 0;
}

File: AlgorithmsToolbox/Week1/APlusB.py
# Uses python3
# There are two ways of running this program:
# 1. Run
#     python3 APlusB.py
# then enter two numbers and press ctrl-d/ctrl-z
# 2. Save two numbers to a file -- say, dataset.txt.
# Then run
#     python3 APlusB.py < dataset.txt

a, b = input().split()
a = int(a)
b = int(b) 

if(a >= 0 and b<=9):
	print(a+b)
else:
	print("Error in inputs")

